test/fixtures/basic/nuxt.config.ts

```diff
@@ -0,0 +1,10 @@
+import MyModule from '../../../src/module'
+
+export default defineNuxtConfig({
+  modules: [
+    MyModule
+  ],
+  cloudinary: {
+    cloudName: 'ddtc5sowb',
```

Comment:
is this your cloud name still? if the tests dont need a real ID i wouldnt both including a real one. just like TESTCLOUD or something if you're just inspecting the URLs / output

Reply:
It is needed for tests that check if the generated src is correct

Reply:
does it actually make a network request though? cause if not the assertion can just include that ID. i wouldnt expect the need to make HTTP requets to the URLs for the tests but thats just my 2c

https://github.com/colbyfayock/next-cloudinary/blob/main/next-cloudinary/tests/loaders/cloudinary-loader.spec.js#L29